_projects/Geodis-Wilson-Configurator.md
---
title: Geodis Wilson Configurator
permalink: /work/geodis-wilson-configurator/
order: 6
projectTitle: "Geodis Wilson Configurator"
support: "ios"
category: "b2b_sort"
image: "2013/10/Feature-image-495x400.png"
link: "geodis-wilson-configurator"
socialShareText: "Geodis+Wilson+Configurator"
---
<div class="avia-layerslider solid_bottom_border">
  <div id="layerslider_1" class="ls-wp-container">

    <div class="ls-layer" style="slidedirection: right; slidedelay: 4000; durationin: 1500; durationout: 1500; easingin: easeInOutQuint; easingout: easeInOutQuint; delayin: 0; delayout: 0; timeshift: 0; transition2d: 24; ">
      <img src="{{ site.assetsurl }}2013/10/BG.png" class="ls-bg" alt="Slide background">
      <img class="ls-s-1" src="{{ site.assetsurl }}2013/10/BG1.png" style="position: absolute; top: 0px; left: 0px; slidedirection : fade; slideoutdirection : fade; durationin : 900; durationout : 900; easingin : easeInQuint; easingout : easeInOutQuint; delayin : 0; delayout : 3000; scalein : 2.0; scaleout : 2.0; showuntil : 0; ">
      <img class="ls-s-1" src="{{ site.assetsurl }}2013/10/iPad-white.png" style="position: absolute; top: 0px; left: 0px; slidedirection : left; slideoutdirection : left; durationin : 500; durationout : 500; easingin : swing; easingout : swing; delayin : 600; delayout : 3100; showuntil : 0; ">
      <img class="ls-s-1" src="{{ site.assetsurl }}2013/10/iPad-black.png" style="position: absolute; top: 0px; left: 0px; slidedirection : left; slideoutdirection : left; durationin : 600; durationout : 600; easingin : easeInOutQuint; easingout : easeInOutQuint; delayin : 600; delayout : 3200; showuntil : 0; ">
      <img class="ls-s-1" src="{{ site.assetsurl }}2013/12/name.png" style="position: absolute; top: 10px; left: 0px; slidedirection : right; slideoutdirection : right; durationin : 500; durationout : 600; easingin : easeInOutQuint; easingout : easeInOutQuint; delayin : 600; delayout : 3300; showuntil : 0; ">
      <img class="ls-s-1" src="{{ site.assetsurl }}2013/12/text-1.png" style="position: absolute; top: 0px; left: 0px; slidedirection : right; slideoutdirection : right; durationin : 800; durationout : 500; easingin : easeInOutQuint; easingout : easeInOutQuint; delayin : 600; delayout : 3400; showuntil : 0; ">
      <img class="ls-s-1" src="{{ site.assetsurl }}2013/12/text-2.png" style="position: absolute; top: 0px; left: 0px; slidedirection : right; slideoutdirection : right; durationin : 800; durationout : 500; easingin : easeInOutQuint; easingout : easeInOutQuint; delayin : 800; delayout : 3400; showuntil : 0; ">
      <img class="ls-s-1" src="{{ site.assetsurl }}2013/12/icon-1.png" style="position: absolute; top: 0px; left: 0px; slidedirection : right; slideoutdirection : right; durationin : 500; durationout : 500; easingin : easeInOutQuint; easingout : easeInOutQuint; delayin : 650; delayout : 3500; showuntil : 0; ">
      <img class="ls-s-1" src="{{ site.assetsurl }}2013/12/icon-2.png" style="position: absolute; top: 0px; left: 0px; slidedirection : right; slideoutdirection : right; durationin : 500; durationout : 500; easingin : easeInOutQuint; easingout : easeInOutQuint; delayin : 700; delayout : 3600; showuntil : 0; ">
      <img class="ls-s-1" src="{{ site.assetsurl }}2013/12/icon-3.png" style="position: absolute; top: 0px; left: 0px; slidedirection : right; slideoutdirection : right; durationin : 500; durationout : 500; easingin : easeInOutQuint; easingout : easeInOutQuint; delayin : 750; delayout : 3700; showuntil : 0; ">
      <img class="ls-s-1" src="{{ site.assetsurl }}2013/12/icon-4.png" style="position: absolute; top: 0px; left: 0px; slidedirection : right; slideoutdirection : right; durationin : 500; durationout : 500; easingin : easeInOutQuint; easingout : easeInOutQuint; delayin : 800; delayout : 3800; showuntil : 0; ">
      <img class="ls-s-1" src="{{ site.assetsurl }}2013/12/icon-5.png" style="position: absolute; top: 0px; left: 0px; slidedirection : right; slideoutdirection : right; durationin : 500; durationout : 500; easingin : easeInOutQuint; easingout : easeInOutQuint; delayin : 850; delayout : 3900; showuntil : 0; ">
      <img class="ls-s-1" src="{{ site.assetsurl }}2013/12/icon-6.png" style="position: absolute; top: 0px; left: 0px; slidedirection : right; slideoutdirection : right; durationin : 500; durationout : 500; easingin : easeInOutQuint; easingout : easeInOutQuint; delayin : 900; delayout : 4000; showuntil : 0; ">
      <img class="ls-s-1" src="{{ site.assetsurl }}2013/12/icon-7.png" style="position: absolute; top: 0px; left: 0px; slidedirection : right; slideoutdirection : right; durationin : 500; durationout : 500; easingin : easeInOutQuint; easingout : easeInOutQuint; delayin : 950; delayout : 4100; showuntil : 0; ">
    </div>
  </div>
</div>

<div class="wrapper content project-detail" markdown="1">
  <h2 class="content-h2 with-bottom-line">Geodis Wilson</h2>

#### Mission

Allow Sales Managers, from global logistics player, Geodis Wilson to conduct multimedia presentations; that are already honed to their client’s needs and requirements.

#### Concept

With a broad portfolio of products developed for a wide range of industry sectors, it was critical to allow the Sales Manager to only showcase relevant solutions and content in their presentations; whether products, PDFs or videos. Through a simple client profiling tool, Sales Managers are able to create tailor made solutions, in advance of their appointment. During the meeting, the Sales Manager can gauge client interest in particular products in more detail. This allows them to wrap up the meet by sending a fine tuned, summary PDF of the presentation to the client, with all relevant products and solutions included. All of the tablet's presentation material can be controlled via a central CMS. This allows the CMS administrator to add, modify and delete all content, and with a press of a button, push the new updates out to all users, thus keeping the Sales Managers up-to-date.

#### Results

Coming soon

</div>

{% include share-and-download.html %}

<script>
$(document).ready(function() {
  if (typeof $.fn.layerSlider == "undefined") {
    lsShowNotice('layerslider_1','jquery');
  }
  else if (typeof $.transit == "undefined" || typeof $.transit.modifiedForLayerSlider == "undefined") {
    lsShowNotice('layerslider_1', 'transit');
  }
  else
  {
    $("#layerslider_1").layerSlider({
      width : '1440px',
      height : '600px',
      responsive : true,
      responsiveUnder : 0,
      sublayerContainer : 0,
      autoStart : false,
      pauseOnHover : true,
      firstLayer : 1,
      animateFirstLayer : true,
      randomSlideshow : false,
      twoWaySlideshow : true,
      loops : 0,
      forceLoopNum : true,
      autoPlayVideos : true,
      autoPauseSlideshow : 'auto',
      youtubePreview : 'maxresdefault.jpg',
      keybNav : true,
      touchNav : true,
      skin : 'fullwidth',
      skinsPath : '../../css/LayerSlider/skins/',
      globalBGColor : '#ffffff',
      navPrevNext : true,
      navStartStop : true,
      navButtons : true,
      hoverPrevNext : true,
      hoverBottomNav : false,
      showBarTimer : false,
      showCircleTimer : true,
      thumbnailNavigation : 'hover',
      tnWidth : 100,
      tnHeight : 60,
      tnContainerWidth : '60%',
      tnActiveOpacity : 35,
      tnInactiveOpacity : 100,
      imgPreload : true,
      yourLogo : false,
      yourLogoStyle : 'left: 10px; top: 10px;',
      yourLogoLink : false,
      yourLogoTarget : '_self',
      cbInit : function(element) { },
      cbStart : function(data) { },
      cbStop : function(data) { },
      cbPause : function(data) { },
      cbAnimStart : function(data) { },
      cbAnimStop : function(data) { },
      cbPrev : function(data) { },
      cbNext : function(data) { }
    });
  }
});
</script>


news.md
---
layout: default
title: Mobile Now Group | News - Mobile Now Group
permalink: /news/
---
hi, this is news page

js/LayerSlider/config.js
$(document).ready(function() {
  if (typeof $.fn.layerSlider == "undefined")
  {
    lsShowNotice('layerslider_1','jquery');
  }
  else if (typeof $.transit == "undefined" || typeof $.transit.modifiedForLayerSlider == "undefined")
  {
    lsShowNotice('layerslider_1', 'transit');
  }
  else
  {
    $("#layerslider_1").layerSlider({
      width : '1440px',
      height : '650px',
      responsive : true,
      responsiveUnder : 0,
      sublayerContainer : 0,
      autoStart : false,
      pauseOnHover : true,
      firstLayer : 1,
      animateFirstLayer : true,
      randomSlideshow : false,
      twoWaySlideshow : true,
      loops : 0,
      forceLoopNum : true,
      autoPlayVideos : true,
      autoPauseSlideshow : 'auto',
      youtubePreview : 'maxresdefault.jpg',
      keybNav : true,
      touchNav : true,
      skin : 'fullwidth',
      skinsPath : 'css/LayerSlider/skins/',
      navPrevNext : true,
      navStartStop : false,
      navButtons : false,
      hoverPrevNext : true,
      hoverBottomNav : false,
      showBarTimer : false,
      showCircleTimer : true,
      thumbnailNavigation : 'disabled',
      tnWidth : 300,
      tnHeight : 60,
      tnContainerWidth : '60%',
      tnActiveOpacity : 35,
      tnInactiveOpacity : 100,
      imgPreload : true,
      yourLogo : false,
      yourLogoStyle : 'position: absolute; left: 10px; top: 10px; z-index: 99;',
      yourLogoLink : false,
      yourLogoTarget : '_self',
      cbInit : function(element) { },
      cbStart : function(data) { },
      cbStop : function(data) { },
      cbPause : function(data) { },
      cbAnimStart : function(data) { },
      cbAnimStop : function(data) { },
      cbPrev : function(data) { },
      cbNext : function(data) { }
    });
  }
});
